demo/index.ts

```diff
@@ -7,7 +7,8 @@ import { Schema } from "prosemirror-model";
 import { schema as basicSchema } from "prosemirror-schema-basic";
 import { EditorState } from "prosemirror-state";
 import { EditorView } from "prosemirror-view";
-import { createImageElement } from "../src/elements/demo-image/DemoImageElement";
+import { createImageElement } from "../src";
+import { codeElement } from "../src/elements/code/CodeElementSpec";
```

Comment:
We'll need to include `codeElement` as an export of src when the time comes to add the element to Composer.